File: backend/aws-api/user-crud/DeleteController.js
exports.remove = function(pool, body, callback) {
    let obj = JSON.parse(body),
        userId = obj["userId"],
        sql = `DELETE FROM users WHERE user_id = ${userId}`;
        
    pool.getConnection(function(err, connection) {
        if(err) throw err;
        connection.query(sql, function(error, results) {
            if(error) {
                callback(error);
            } else {
                const result = {
                    'statusCode': 200,
                    'headers': {
                        'Content-Type': 'application/json',
                        'Access-Control-Allow-Headers': 'Content-Type',
                        'Access-Control-Allow-Origin': '*',
                        'Access-Control-Allow-Methods': 'OPTIONS,DELETE'
                    },
                    'body': "Delete user success"
                };
                callback(null, result);
            }
        });
    });
};

File: backend/aws-api/tree-crud/DeleteController.py
import json

def delete(connection, data):
    cursor = connection.cursor()
    body = json.loads(data)
    
    sql = "DELETE FROM trees WHERE tree_id = %s" % body['treeID']
    
    statusCode = 502
    message = "Delete tree record failure"
    try:
        cursor.execute(sql)
        connection.commit()
        statusCode = 200
        message = "Delete tree record succeeded"
    except:
        connection.rollback()

    # print(rows)
    return {
        'statusCode': statusCode,
        'headers': {
            'Content-Type': 'application/json',
            'Access-Control-Allow-Headers': 'Content-Type',
            'Access-Control-Allow-Origin': '*',
            'Access-Control-Allow-Methods': 'OPTIONS,DELETE'
        },
        'body': message
    }